### PR TITLE
paramminer: bug squashing

### PR DESCRIPTION
**Pull request by ArkaprabhaChakraborty**
Signed-off-by: ArkaprabhaChakraborty <chakrabortyarkaprabha998@gmail.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
Why is this being pushed to a PR before it’s all sorted out?

**Comment by ArkaprabhaChakraborty** [comment]
Uhh so that @thc202 and everyone can take a look :)

**Comment by ArkaprabhaChakraborty** [comment]
I'll merge every commits and do the clean up :)

**Comment by ArkaprabhaChakraborty** [comment]
should have made it a draft actually :)

**Comment by kingthorin** [comment]
Sorry for the rant, woke up to a ton of notifications on print statements. It’s fine I’m getting coffee now 🤪

**Comment by ArkaprabhaChakraborty** [comment]
Good Morning!!! 🙃 🙃 🙃

**Comment by ArkaprabhaChakraborty** [comment]
@kingthorin @psiinon @thc202

File: addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/UrlBruteForce.java [hunk 1]
```diff
@@ -297,39 +302,39 @@ public String requester(HttpMessage msg, Method method, Map<String, String> para
...
-            return new ParamReasons("word count mismatch", params);
+            return new ParamReasons(Reason.WORD_COUNT, params);
         }
 
-        if (resp1.compareWith(resp2) < 1) {
-            return new ParamReasons("body mismatch", params);
-        }
+        // if (resp1.compareWith(resp2) < 1) {
+        //     return new ParamReasons("body mismatch", params);
+        // }
```
Comment (kingthorin) [inline addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/UrlBruteForce.java:332]: Is this block in or out? (Or is that still up for testing/future PR)

**Review by kingthorin — APPROVED** [review]

File: addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/UrlBruteForce.java (hunk 1 above)
Comment (thc202) [reply addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/UrlBruteForce.java:332]: Future PR.

Comment (kingthorin) [reply addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/UrlBruteForce.java:332]: 👍

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**